pkg/logcli/output/jsonl_test.go

```diff
@@ -0,0 +1,72 @@
+package output
+
+import (
+	"testing"
+	"time"
+
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestJSONLOutput_Format(t *testing.T) {
```

Comment:
maybe test whether this is even valid json

Reply:
Good idea, done.